docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-indexes-syntax.md

```diff
@@ -0,0 +1,8 @@
+# SHOW INDEXES 语法
+
+```ngql
+SHOW {TAG | EDGE} INDEXES
+```
+
+`SHOW INDEXES` 用于列出已创建完成的标签或边类型的索引信息。`SHOW INDEXES` 返回以下字段：索引 ID 和 索引名称。
```

Comment:
add example

Reply:
I'm migrating docs to another [repo](https://github.com/vesoft-inc/nebula-docs). Will fix there.